src/reducers/entities/preferences.js

```diff
@@ -10,6 +10,7 @@ function getKey(preference) {
 
 function myPreferences(state = {}, action) {
     switch (action.type) {
+    case PreferencesTypes.SAVE_PREFERENCES:
```

Comment:
why the reducer does the same when saving optimistically and when committing? I get that `RECEIVED_PREFERENCES` is called whenever we call `getPreferences` but this is adding the same preference twice once when saving and once on commit.

Also I thought we discuss that for `getMyPreferences` we were going to set the state to whatever was returned from the server.

I think im somehow lost here, care to clarify?